runtime/near-vm/.github/workflows/test-sys.yaml

```diff
@@ -0,0 +1,100 @@
+name: Runtime tests
```

Comment:
Does this get run at all given that this `.github` is not top-level? Double-check if testing occurs for this crate in buildkite and remove this file.